.github/workflows/coverage.yml

```diff
@@ -15,16 +16,20 @@ jobs:
           profile: minimal
           toolchain: nightly
           override: true
-      - uses: actions-rs/cargo@v1
+          components: llvm-tools-preview
+      - uses: actions-rs/install@v0.1
         with:
-          command: test
-          args: --all-features --no-fail-fast
-        env:
-          CARGO_INCREMENTAL: "0"
-          RUSTFLAGS: "-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests"
-      - id: grcov
-        uses: actions-rs/grcov@v0.1
-      - uses: coverallsapp/github-action@v1
+          crate: cargo-binutils
+          version: 0.3.3
+      - uses: actions-rs/install@v0.1
+        with:
+          crate: rustfilt
+          version: 0.2.1
+      - name: Run test coverage
+        run: |
+          chmod +x .github/workflows/scripts/coverage.sh
+          .github/workflows/scripts/coverage.sh
```

Comment:
can't we just run `bash .github/workflows/scripts/coverage.sh` instead of chmod?

Reply:
I think this is a hold-out from my local changes where the script didn't have executable permission. The file in the repo should have them now, so this can be removed.